src/main/scala/scalafix/internal/sbt/ScalafixCoursier.scala

```diff
@@ -44,24 +45,27 @@ object ScalafixCoursier {
   private val dependencyCache: jutil.Map[Seq[ModuleID], List[Path]] = {
     jutil.Collections.synchronizedMap(new jutil.HashMap())
   }
-  private[scalafix] def fetchScalafixDependencies(resolvers: Seq[Repository])
-    : function.Function[Seq[ModuleID], List[Path]] =
-    (t: Seq[ModuleID]) => {
-      val dependencies = t.map { module =>
-        val binarySuffix =
-          if (module.crossVersion.isInstanceOf[CrossVersion.Binary]) "_2.12"
-          else ""
-        new Dependency(
-          module.organization,
-          module.name + binarySuffix,
-          module.revision
+  private[scalafix] def fetchScalafixDependencies(
+      resolvers: Seq[Repository]
+  ): function.Function[Seq[ModuleID], List[Path]] =
+    new function.Function[Seq[ModuleID], List[Path]] {
+      override def apply(t: Seq[ModuleID]): List[Path] = {
+        val dependencies = t.map { module =>
+          val binarySuffix =
+            if (module.crossVersion.isInstanceOf[CrossVersion.Binary]) "_2.12"
+            else ""
+          new Dependency(
+            module.organization,
+            module.name + binarySuffix,
+            module.revision
+          )
+        }
+        CoursierSmall.fetch(
+          fetchSettings
+            .withRepositories(resolvers ++: fetchSettings.repositories)
```

Comment:
Note that this breaks a standard sbt behaviour of `scalafixResolvers`: with `scalafixResolvers := Nil` we will still end up with resolver from `fetchSettings.repositories`.

Reply:
How about renaming the setting to `scalafixCustomResolvers`? I don't see a valid use-case to overwrite the base resolvers.

Reply:
FYI, I kickstarted a release for coursier-small v1.2.0 https://travis-ci.org/olafurpg/coursier-small/builds/458497906 should be on Maven within the next hr